lib/gpio/pin.toit

```diff
@@ -142,9 +143,136 @@ class Pin:
       finally: | is-exception _ |
         if is-exception: close
 
+  /**
+  Opens a GPIO Pin on the chip identified by the given $path and $num (often called "offset").
```

Comment:
It looks like we use Pin rather than pin here in the text in a pretty consistent way. I think I'd prefer pin or $Pin.

Reply:
done. (changed to 'pin' consistently).